iwharfclientfetcher.go

```diff
@@ -1,7 +1,6 @@
 package main
 
-import "github.com/iver-wharf/wharf-api-client-go/pkg/wharfapi"
-
+/*
 type wharfClientAPIFetcher interface {
```

Comment:
> @Alexamakans 
> * The testing code has not been updated and is therefore not functional currently.
>   If we want to keep it the way it is (updated so it works) then I can spend time updating it.
>   Otherwise I will remove it, and we can come back to it when we have decided on how we want to write our tests.
>   **Should we spend time updating it, or remove it for now?**

We've to update it. I've not read all of the tests so I don't know how big of a task this is but **even** if the long-term plan is to remove all of this code later when hiding it behind the wharf-api, that's probably still months away.

From a glance it seems the biggest issue is the `wharfClientAPIFetcher` in this file that needs to be updated, which is perhaps no small feat as no known refactoring tool can aid you in this. If you want we can split the task, or do a Live Code session in VS Code where we refactor it together.